lib/Haskell/Prim.agda

```diff
@@ -53,11 +53,11 @@ f $ x = f x
 -- Language constructs
 
 infix -1 case_of_
-case_of_ : a → (a → b) → b
+case_of_ : {@0 a b : Set ℓ} → (a' : a) → ((a'' : a) → @0 {{ a' ≡ a'' }} → b) → b
```

Comment:
Why is it necessary to add the quantification `{@0 a b : Set ℓ} → ` here?

Reply:
Ah, I mindlessly copied this from the `if_then_else_` solution. It is removed now!